docs/application/native/guides/multimedia/media-conversions.md

```diff
@@ -1,18 +1,14 @@
 # Media Conversions
 
 
-You can perform various media conversions through codecs. You can encode and decode video and audio data, and, in mobile applications, you can also use video transcoding features, such as multi-format codec and output format.
+You can perform various media conversions through codecs. You can encode and decode video and audio data.
 
-The main features of the Media Codec and Video Util API include:
+The main features of the Media Codec API include:
```

Comment:
Since the API name and feature name are the same, can we remove the following lines:

The main features of the Media Codec API include:
- Media codecs